docs/docsite/requirements.txt

```diff
@@ -1,13 +1,16 @@
-#pip packages required to build docsite
+# pip packages required to build docsite
+# these requirements are as loosely defined as possible
+# if you want known good versions of these dependencies
+# use known_good_reqs.txt instead 
 jinja2
 PyYAML
 rstcheck
-sphinx==2.1.2
+sphinx
 sphinx-notfound-page >= 0.6
 sphinx-intl
-sphinx_ansible_theme === 0.6.0
```

Comment:
I'd like to have an exact pin here. `==` matches `.post` releases while `===` matches the exact spec.

Reply:
Ah, thanks @webknjaz, that is not syntax I knew before. I'll restore that.

Reply:
It's documented here: https://www.python.org/dev/peps/pep-0440/#arbitrary-equality